src/Input/InputAction.js

```diff
@@ -0,0 +1,53 @@
+// @flow weak
+
+import React from 'react';
+import type { Node, ElementType } from 'react';
+import classNames from 'classnames';
+import withStyles from '../styles/withStyles';
+
+export const styles = (theme: Object) => ({
+  root: {
+    position: 'absolute',
+    right: -theme.spacing.unit * 2,
+    top: theme.spacing.unit,
+  },
+});
+
+type Default = {
+  classes: Object,
+};
+
+export type Props = {
+  /**
+   * The content of the component, normally an `IconButton`.
+   */
+  children?: Node,
+  /**
+   * Useful to extend the style applied to components.
+   */
+  classes?: Object,
+  /**
+   * @ignore
+   */
+  className?: string,
+  /**
+   * 
+   */
+  component?: ElementType,
+};
+
+function InputAction(props: Default & Props) {
+  const { children, component, classes, className, ...other } = props;
+
+  const Component = component || 'div';
```

Comment:
React provides a mechanism for default, let's use it, it automatically added in the docs.